ultra/tests/test_train.py

```diff
@@ -41,6 +42,21 @@ class TrainTest(unittest.TestCase):
     # Put generated files and folders in this directory.
     OUTPUT_DIRECTORY = "tests/train_test/"
 
+    @classmethod
+    def setUpClass(cls):
+        """Generate model"""
+        model_log_dir = os.path.join(TrainTest.OUTPUT_DIRECTORY, "model_logs/")
+        try:
+            os.system(
+                f"python ultra/train.py --task 00 --level easy --episodes 3 --eval-rate 2 --max-episode-steps 2 --log-dir {model_log_dir} --eval-episodes 1 --headless"
+            )
+        except Exception as err:
+            print(err)
+            self.assertTrue(False)
+
+        if not os.path.exists(model_log_dir):
+            self.assertTrue(False)
```

Comment:
`self` doesn't seem to be an attribute of this method. I wonder if we can use this setup method to just call a multi-agent training run and a single agent training run, and then all tests that need a training run, can just use the experiment directories that were created by this method (like how `test_evaluate.py` does it).

I can take a look at this too, but let me know if you have any ideas.